inventory/group_vars/avr/vars.yml

```diff
@@ -0,0 +1,12 @@
+nbofvms: 1
```

Comment:
Do we need this file?

Reply:
"group_vars/{{ env }}/vars.yml" contains variables that are SPECIFIC to a particular environ.
Example of specific var's:
PLAT: a User/Pass can be different for SP and for VNV
POP: Openstack variables (ex, 'flavor') are different on each PoP
ENV: variables to distinguish the different environ's